src/mbgl/map/map.cpp

```diff
@@ -760,6 +756,18 @@ std::vector<Feature> Map::queryRenderedFeatures(const ScreenBox& box, const opti
     });
 }
 
+AnnotationIDs Map::queryPointAnnotations(const ScreenBox& box) {
+    auto features = queryRenderedFeatures(box, {{ AnnotationManager::PointLayerID }});
+    AnnotationIDs ids;
+    ids.reserve(features.size());
+    std::transform(features.begin(), features.end(), std::back_inserter(ids), [](auto& feature) -> AnnotationID {
```

Comment:
We generally prefer a range-based for loop + `push_back` over `std::transform` + `std::back_inserter`.